web/src/app/schedules/ScheduleShiftList.tsx

```diff
@@ -117,6 +117,85 @@ function ScheduleShiftList({
     },
   })
 
+  function getShiftDetails(
```

Comment:
it would be helpful to have a quick one line comment giving the gist of the purpose of the function.